app/views/videos/show.html.haml

```diff
@@ -7,4 +8,35 @@
         %header
           %h3= @video.title # No need for string interpolation as the expression returns a string anyways
```

Comment:
Good point, although I could say the same thing about your comment in the code ;). I would treat this as you would a professional commit. By that I mean communicate to  your team (the TA's) through line notes (this is a line note) vs comments in the code. If you have something that is strictly related to the code base a comment is fine (but generally eschewed in ruby) but if you have something that is more team level (you are contrasting your code with the solution given) do it at the line note level. The big difference is that the comment sticks around in the codebase like a tattoo that was an inside joke with your ex, when you really should have kept around a ceramic reminder that you can shatter on the floor or pack in *their* box of stuff (depending on the level of drama involved in the breakup).